### ISSUE TITLE
Improve colour contrast of Message above Search form

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/FreeCENMigration#1105]
The text above the search form is:
You may access the original version of Freecen by clicking [this link]

This is not helpful for a screen-reader user (who can call up a list of links on a page) because 'this link' could be anything.
Also, the casing of 'Freecen' should be 'FreeCEN'.

Suggest that the text would be better as:
You can use this link to visit the [original version of FreeCEN]

**Comment by AlOneill** [comment]
Message edited on production and on test3.

**Comment by AlOneill** [comment]
The colour contrast between the plain (red) text and the (lavender) link is an extremely poor **1.01:1** (both colours have good contrast with the white background, but were not intended to be used side-by-side). Also, the whole Message is in bold, so there is not even a contrast in font weight between the plain text and the link text.

The text colour and weight are set with a class on the Message container (overriding the normal weight and default dark grey), which is not in Refinery, so a developer would have to make any permanent change. This would be a good idea to avoid contrast issues in the future for Messages.

In the meantime, we can override the class in Refinery. But what would be a good alternative?

We could simply return the plain text to the default grey and its weight to normal.

Will edit title of story and move it to Needs Review so that we can **discuss the options for a more accessible colour scheme**.

**Comment by PatReynolds** [comment]
@AlOneill to tweak the colours in refinery, with the developers to use once we have a good solution.

Suggestions: changing colours, red box islet.

NB currently used to redirect to FreeCEN1, could be used for other messages with link

**Comment by DeniseColbert** [comment]
Consider islet or island class for message of the day where we want to draw attention. This message can be changed to plain text in grey (from red). @AlOneill to make the change on test3,

**Comment by AlOneill** [comment]
"Message" edited on test3 and is ready for Review. This is the simplest version.

Stick or twist?

**Comment by DeniseColbert** [comment]
Stick; looks good.

**Comment by AlOneill** [comment]
This is not complete yet — the  `weight--bold` and `text--red` classes should be removed from the `div` element that contains the "Message"  — a developer task.

**Comment by AlOneill** [comment]
Then "Message" should be cleaned up on test3 and either synced or copied to live — I can do that.

**Comment by Captainkirkdawson** [comment]
The div has been adjusted prior to deployment and it now operational.
Please proceed with any adjustments to the refinery content

**Comment by AlOneill** [comment]
@Captainkirkdawson, thanks — I can now see the changes.

"Message" cleaned up on t3 and live.

**Comment by Captainkirkdawson** [comment]
TY
